src/main.rs

```diff
@@ -34,55 +34,100 @@ async fn main() -> Result<(), Box<dyn Error>> {
     let mut clusters = HashMap::new();
 
     let password = match opt.password {
-        true => rpassword::read_password_from_tty(Some("Password: ")).unwrap(),
-        false => String::from("password"),
+        true => Some(rpassword::read_password_from_tty(Some("Password: ")).unwrap()),
+        false => None,
     };
 
     let active = if config.clusters().is_empty() {
         let timeouts = ClusterTimeouts::default().export_lcb_args();
+        let hostnames = if let Some(hosts) = opt.hostnames {
+            hosts
+        } else {
+            "localhost".into()
```

Comment:
nit: maybe make localhost, administrator etc const values at the top of the file? makes it easier to change/reuse later